apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx

```diff
@@ -1,10 +1,13 @@
 import { isFuture } from 'date-fns';
-import { useState } from 'react';
+import React, { useState } from 'react';
```

Comment:
question (non-blocking): why is the React import needed here?

Reply:
My IDE sometimes does this automatically. I'll remove it the next time I'm working on the project. Thanks